.gitlab/source_test/ebpf.yml

```
@@ -88,16 +77,52 @@ tests_ebpf_x64:
   tags: ["runner:main"]
   variables:
     ARCH: amd64
+    TASK_ARCH: x64
 
 tests_ebpf_arm64:
   extends: .tests_linux_ebpf
   image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/system-probe_arm64:$DATADOG_AGENT_SYSPROBE_BUILDIMAGES
   tags: ["runner:docker-arm", "platform:arm64"]
   variables:
     ARCH: arm64
+    TASK_ARCH: arm64
+
+.prepare_ebpf_tests:
+  stage: source_test
+  needs: ["go_deps"]
+  artifacts:
+    when: always
+    paths:
+      - $CI_PROJECT_DIR/.tmp/binary-ebpf
+      - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-security-agent-check/files
+      - $DD_AGENT_TESTING_DIR/site-cookbooks/dd-system-probe-check/files
+  before_script:
+    - !reference [.retrieve_linux_go_deps]
+    - mkdir -p $CI_PROJECT_DIR/.tmp/binary-ebpf/co-re
+    - !reference [.retrieve_sysprobe_deps]
+  script:
+    - !reference [.build_sysprobe_artifacts]
```

```suggestion
    - inv -e install-tools
    - inv -e system-probe.object-files
    - !reference [.build_sysprobe_artifacts]
```
I think you need  `install-tools` to get `gotestsum`, and `system-probe.object-files` because `kitchen-prepare` doesn't build the eBPF object files, which are needed by `build_sysprobe_artifacts`